src/pages/NanoBrowserQuest/Guide/guide.md

```diff
@@ -239,9 +307,11 @@ Your character will receive bonuses by wearing all set items
 ## Other
 
 **1. Cape**
-Attributes: Defense + Random - Attack/Defence/EXP when in a party
-Dropped by: Sold by the Red Wizard in the village and also dropped by Minotaur
-Droprate:
+Attributes: Defense (1) + Either Attack/Defence/EXP % when player is in a party. The second attribute is randomly decided when the items is bought/dropped
+Dropped by (Droprate): Sold by the Red Wizard in the village and also dropped by Minotaur (?)
```

Comment:
The Mino doesn't drop it but there is a 3% chance of getting it in the blue chest